packages/react-native-win32/src/Libraries/Utilities/FocusManager.win32.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import * as React from 'react';
+import {findNodeHandle, NativeModules} from 'react-native';
+
+/*
+ ** This is a helper class intended to allow usage of Polite/Aggressive Focus for win32.
+ */
+class FocusManager {
+  // This function takes in a ref to a React Component and a bool value. If setWindowFocus = true, call aggressive focus.
+  // Else, call polite focus
+
+  static focus(ref: React.Ref<any>, setWindowFocus: boolean) {
+    if (ref) {
+      if (setWindowFocus) {
+        NativeModules.UIManager.dispatchViewManagerCommand(
+          findNodeHandle(ref),
+          NativeModules.UIManager.getViewManagerConfig('RCTView').Commands
+            .aggressivefocus,
+          null,
+        );
+      } else {
+        NativeModules.UIManager.dispatchViewManagerCommand(
```

Comment:
Do you actually need two separate commands, why not just a single one that takes the arg? Unless something has changed in newer releases, you can pass the args as an object in place of the null you currently use

Reply:
I think, if for semantic reasons, you want to expose polite and aggressive focus via different functions that should only manifest at the highest levels of the stack. The JS functions should use the same code in impls, so native side additions are minimal.

Reply:
Given that we wrote a static FocusManager class I would've preferred to see this on a NativeModule instead of forking the ViewManager config further but I'm assuming you already checked in the native code for this?

Reply:
I guess this is probably fine.  We'll likely need to reconcile all this stuff when Fabric rolls around anyway.